.github/workflows/kind-e2e.yaml

```diff
@@ -50,6 +50,10 @@ jobs:
       DAPR_REGISTRY: localhost:5000/dapr
       DAPR_TAG: dev
       DAPR_NAMESPACE: dapr-tests
+      # Useful for upgrade/downgrade/compatibility tests
```

Comment:
Please update dapr-test.yaml too (for AKS)

Reply:
I am not adding it there because it does not work on Windows' latest version since the old image does not have an image for that.

Reply:
How about adding it for the Linux tests only?

Reply:
This does the same. See the comments next to it. The goal is to add it to work with both once we have the new os build available in the old images.